src/utils.py

```diff
@@ -291,6 +291,44 @@ def contains_point(self, x: float, y: float) -> bool:
         return (x - self.x)**2 + (y - self.y)**2 <= self.radius**2
 
 
+@dataclass(frozen=True)
+class Triangle(_Geom2D):
+    """A helper class for visualizing and collision checking triangles."""
```

Comment:
@ronuchit you're not going to be able to use this class for collision checking without implementing some collision checking code -- update docstring?

Reply:
i was thinking of contains_point as doing some crude collision checking (and that's all i'm gonna use in satellites anyway). let's just leave it as is